src/fmu/dataio/export/rms/_utils.py

```diff
@@ -45,9 +50,18 @@ def _check_rmsapi_version(minimum_version: str) -> None:
     logger.debug("Check API version... DONE")
 
 
-def _get_rms_project_units(project: Any) -> str:
+def get_rms_project_units(project: Any) -> str:
     """See if the RMS project is defined in metric or feet."""
 
     units = project.project_units
     logger.debug("Units are %s", units)
     return str(units)
+
+
+def load_global_config() -> dict[str, Any]:
+    if not CONFIG_PATH.exists():
+        raise RuntimeError(
```

Comment:
`FileNotFoundError` could work here as well and might be more explanatory to users